src/main/java/vg/civcraft/mc/civmodcore/CivModCorePlugin.java

```diff
@@ -1,26 +1,24 @@
 package vg.civcraft.mc.civmodcore;
 
+import java.util.logging.Logger;
+
+import org.bukkit.plugin.java.JavaPlugin;
+
 /**
  * The sole purpose of this class is to make Spigot recognize this library as a plugin and automatically
  * load the classes onto the classpath for us.
  * <p>
  * Replaces Dummy class.
  */
-public class CivModCorePlugin extends ACivMod {
-
-	@Override
-	public void onLoad() {
-		//Don't want it to load config
-	}
-
-	@Override
-	public void onEnable() {
-		//Don't want it to load commands/events
-	}
+public class CivModCorePlugin extends JavaPlugin {
 
-	@Override
-	protected String getPluginName() {
-		return "CivModCore";
+	/**
+	 * Returns the logger for this plugin. Assumes the plugin has already been loaded.
+	 *
+	 * @return The logger for this plugin.
+	 */
+	public static Logger log() {
```

Comment:
Wouldnt it be better to get the logger from the static singleton reference that every plugin has anyway? Also what do you think of adding a getInstance() method to ACivMod?